tests/test_csg_cad_circulartorus.py

```diff
@@ -0,0 +1,65 @@
+from model_benchmark_zoo import Circulartorus
+import openmc
+import math
+

```

Comment:
CI tests are failing with NameError: name 'np' is not defined, this should fix the error

```suggestion
import numpy as np
```